sql-prepared-plan-cache.md

```diff
@@ -17,6 +17,78 @@ When the execution plan cache is enabled, in the first execution every `Prepare`
 
 TiDB also supports execution plan caching for some non-`PREPARE` statements, similar to the `Prepare`/`Execute` statements. For more details, refer to [Non-prepared plan cache](/sql-non-prepared-plan-cache.md).
 
+## Session-level and instance-level Prepared Plan Cache
+
+TiDB supports both session-level plan cache and instance-level plan cache. Session-level plan cache maintains a separate cache for each session, while instance-level plan cache maintains a single shared cache across all sessions in an instance.
+
+To enable or disable instance-level plan cache, you can use the [`tidb_enable_instance_plan_cache`](/system-variables.md#tidb_enable_instance_plan_cache-new-in-v840) system variable. When you set this variable to `ON`, instance-level plan cache is used. Otherwise, session-level plan cache is used.
```

Comment:
The initial size of instance plan cache is small. We'd better mention setting proper cache size, and how the purging works. 

![image](https://github.com/user-attachments/assets/a797deff-d67b-4fa9-ad84-f8aee5208506)